### PR TITLE
add filter function

### PR DESCRIPTION
**Pull request by shrocket**
The instructions were unclear: Should we have written the filters in App.js and then somehow passed them to List.js? We wrote the filters in List.js, because they worked there better. In the preview, we now see both lists, separated. We assume we should only see one list at a time, but nothing is currently telling us the weather (next step!) . . .

### CODE REVIEW

**Comment by PZhu888** [comment]
LGTM !

File: src/components/List.js [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+  const isSunny = true;
+  const sunnyList = activities.filter(
+    (activity) => activity.isSunnyWeather === isSunny
+  );
+  const rainyList = activities.filter(
+    (activity) => activity.isSunnyWeather !== isSunny
+  );
+
+  const listItems1 = sunnyList.map((activity) => (
+    <>
```
Comment (mbosselmann) [inline src/components/List.js:13]: You do not need fragments. :)

File: src/components/List.js [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+      <li key={activity.id}>
+        {activity.name}
+        {activity.isSunnyWeather ? "😎" : "🌧️"}
+      </li>
+    </>
+  ));
+
+  const listItems2 = rainyList.map((activity) => (
+    <>
```
Comment (mbosselmann) [inline src/components/List.js:22]: Same here.

**Review by mbosselmann — COMMENTED** [review]
Hi Shawn and Ping,
for now it is enough to have one list that renders `goodWeatherActivities` when `isSunny` is `true` or `false`. And if you change the `isSunny` manually to `false` or `true` then it should render the correct activities. You can work with two lists if you want to, but it is not necessary as in the next step the switch will then happen depending on the weather information from the API.  :)
I hope this makes it clearer now. Your code looks good! 👍

**Pull request merged**